src/main/java/com/tratif/pdfgen/document/builders/ParameterBuilder.java

```diff
@@ -15,6 +15,12 @@
  */
 package com.tratif.pdfgen.document.builders;
```

Comment:
hmm, we should keep 2 implementations of a common interface: one for wkhtml2pdf and one for chrome (the former might be not used until we fix other issues). Right now you are just replacing it, we shouldn't waste the previous implementation